src/functions/edi/inbound/handler.ts

```diff
@@ -25,7 +28,7 @@ import {
 const buckets = bucketsClient();
 
 export const handler = async (
-  event: unknown
+  event: TransactionEvent
```

Comment:
nit: should this stay `unknown` here and then parse using the schema in the handler?

Reply:
We [do parse already](https://github.com/Stedi-Demos/bootstrap/pull/125/files#diff-8358646551c8d5bf18791ef3b770c0f60b55e1c8465760cc41d6702f8e6bbf5aR39) - I guess keeping it as `unknown` is actually more accurate.